C7GameData/SaveFormat.cs

```diff
@@ -32,9 +33,10 @@ public class C7SaveFormat
 		public C7SaveFormat()
 		{
 			GameData = new GameData();
+			Rules = new C7RulesFormat();
```

Comment:
Ah, now I see the comment on line 27 about rules being an equivalent to BIC/X/Q.  That is probably one of the ones you were referring to.

I'm kind of thinking that separation is an unnecessary holdover from Civ.  Civ3 embeds a BIQ within a SAV, and that leads to weird situations such as separate TILE and CITY objects, with different structures, in the embedded BIQ and in the "save" part of the SAV.  This was added in the pre-Aztec days and hasn't been used, so I wouldn't take it as gospel.